specs/protocol/stage-1.md

```diff
@@ -0,0 +1,44 @@
+# Stage 1 Roles and Requirements
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+- [Permissionless Fault Proofs](#permissionless-fault-proofs)
+- [Roles for Stage 1](#roles-for-stage-1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+This document outlines configuration requirements (including roles and other parameters)
+for an implementation of the OP Stack to satsify the Stage 1 decentralization requirements as defined
+by
+L2 Beat [[1](https://medium.com/l2beat/introducing-stages-a-framework-to-evaluate-rollups-maturity-d290bb22befe), [2](https://medium.com/l2beat/stages-update-security-council-requirements-4c79cea8ef52)].
+
+## Permissionless Fault Proofs
+
+Stage 1 requires a chain to be operating with Permissionless Fault Proofs.
+
+## Roles for Stage 1
+
+Within the context of an OP Stack, the following roles are required for Stage 1:
+
+1. **Upgrade Controller:** ALthough named for its ability to perform an upgarde, more generally this
```

Comment:
Is "Upgrade Controller" a new term we want to standardize around? If so, we will need to update this in a lot of places for consistency since it's "L1 ProxyAdmin Owner" everywhere (standard config, docs, superchain registry, code, etc.)

Or is this intended to be an "abstract" role that exists in the context of L2 Beat requirements even though we don't have a role with this name in the system?

Reply:
For now I would like to just stealthily introduce this term into the lexicon without attempting to replace the current L1 Proxy Admin owner, though I do think we should try to do that over time (I'll add that to the roles design review I have in draft).

As you'll see in the next PR, we still refer to the 2 of 2 safe with the existing language.